### PR TITLE
Moved the test cases for csi from skip to working.

### PR DESCRIPTION
**Pull request by abdasgupta**
As CSI support is there in AWS and GCP for gardener, CSI focused test cases are added for Gardener conformance test

### CODE REVIEW

**Review by schrodit — APPROVED** [review]

File: integration-tests/e2e/kubetest/description/1.19/skip.json [hunk 1]
```diff
@@ -493,182 +493,5 @@
...
-  { "testcase": "[sig-storage] PersistentVolumes-local  [Volume type: dir] Set fsGroup for local volume should set different fsGroup for second pod if first pod is deleted"},
-  { "testcase": "[sig-api-machinery] AdmissionWebhook Should honor timeout"},
-  { "testcase": "[sig-storage] Pod Disks schedule pods each with a PD, delete pod and verify detach [Slow] for read-only PD with pod delete grace period of \"immediate (0s)\""},
-  { "testcase": "[sig-storage] Zone Support Verify a pod is created on a non-Workspace zone and attached to a dynamically created PV, based on the allowed zones and storage policy specified in storage class"},
-  { "testcase": "[sig-storage] PersistentVolumes-local  [Volume type: dir-link] Set fsGroup for local volume should set different fsGroup for second pod if first pod is deleted"},
-  { "testcase": "[sig-storage] EmptyDir wrapper volumes should not cause race condition when used for git_repo [Serial] [Slow]", "description": "Fails on GL (not investigated why), but test looks for a race condition in K8S which was fixed in K8S v1.9.x and uses GitRepo type volumes which are deprecated since K8S v1.11.0" },
```
Comment (dguendisch) [inline integration-tests/e2e/kubetest/description/1.19/skip.json:672]: I'm not sure why this line (and the next one) were removed from the skip-list? Did you really try to rerun them and they now work?
(also they are not on the working.json, so probably you deleted them by accident?)

**Review by dguendisch — CHANGES_REQUESTED** [review]
Besides a line comment, I'm curious about two things:
1. Did you try the tests on all cloudproviders? (we use the skip and working lists for all cloudproviders together, so would need to exclude certain tests if they don't work on a certain cloudprovider)
2. You also moved a bunch of tests that are storage related but not CSI related. Do these tests now work as well?

File: integration-tests/e2e/kubetest/description/1.19/skip.json (hunk 1 above)
Comment (abdasgupta) [reply integration-tests/e2e/kubetest/description/1.19/skip.json:672]: @dguendisch I ran the tests in was, gcp, openstack with the following command `go run -mod=vendor ./integration-tests/e2e --debug=true --k8sVersion=1.19.2 --cloudprovider=<cloudprovider> --testcasegroup="conformance"`

The tests ran fine. Yes, I accidentally deleted two test cases in working.json and forgot to move them into skip.json. I updated the PR.

Comment (dguendisch) [reply integration-tests/e2e/kubetest/description/1.19/skip.json:672]: Hmm, `testcasegroup="conformance"` will however invoke the conformance tests, not the tests that were moved in this PR!
I'll test it out myself, hopefully next week and can give then better feedback.

**Pull request closed without merging**